pkg/migration/runner_test.go

```diff
@@ -2324,9 +2329,19 @@ func TestDeferCutOver(t *testing.T) {
 		DeferCutOver:         true,
 	})
 	assert.NoError(t, err)
-	err = m.Run(context.Background())
-	assert.Error(t, err)
-	assert.ErrorContains(t, err, "timed out waiting for sentinel table to be dropped")
+	wg := sync.WaitGroup{}
+	wg.Add(1)
```

Comment:
Standard way to doing this:
```go
go f() {
    wg.Add(1)
    defer wg.Done()
}
```

Reply:
There's a race in this suggestion though. It could reach wg.Wait() before it reaches wg.Add(1).

Reply:
The linter also detects the race. So I will change the defer line, but not the `wg.Add()`